bigtable/google/cloud/bigtable/row.py

```diff
@@ -243,7 +243,8 @@ def _delete_cells(self, column_family_id, columns, time_range=None,
 def _retry_commit_exception(exc):
     if isinstance(exc, grpc.RpcError):
         exc = exceptions.from_grpc_error(exc)
-    return isinstance(exc, exceptions.ServiceUnavailable)
+    return (isinstance(exc, exceptions.ServiceUnavailable) or
```

Comment:
You can just do `isinstance(exc, (exceptions.ServiceUnavailable, exceptions.DeadlineExceeded))`.